packaging/installer/kickstart.sh

```diff
@@ -421,6 +421,8 @@ get_system_info() {
             ;;
           rhel)
             DISTRO_COMPAT_NAME="centos"
+          rocky)
```

Comment:
This should just be merged with the above case, using `rocky|rhel` as the matching expression.

Reply:
@maneamarius that is not fixed